backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -26,6 +26,29 @@ include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
 include::modules/oadp-using-data-mover-for-csi-snapshots.adoc[leveloffset=+1]
 
+[id="oadp-cleaning-up-after-data-mover-1-1-backup"]
+== Cleaning up after a backup using Data Mover with OADP 1.1._x_
+
+For OADP 1.1._x_, you must perform a data cleanup after you perform a backup using any version of Data Mover.
+
+The cleanup comprises deleting:
+
+* Snapshots in a bucket
+* Cluster resources
+* VSBs after a backup procedure that is either run by a schedule or is run repetitively
```

Comment:
A fix for this issue will be released soon for OADP 1.1.x

[src](https://access.redhat.com/articles/5456281#-note-datamover-backups-in-oadp-11-requires-vsb-cleanup-when-run-via-a-schedule-or-repetitively--34:~:text=A%20fix%20for%20this%20issue%20will%20be%20released%20soon%20for%20OADP%201.1.x)

@weshayutin do you want this temporary issue to be embedded in official docs?

Reply:
from slack
> we need to remove "A fix for this issue will be released soon for OADP 1.1.x"

resolved.

Reply:
@kaovilai If I am reading your comments correctly, you actually do not want to to mention a fix being added, soon, correct?

Reply:
The issue will not be fixed in OADP 1.1.x, only in OADP 1.2.x

Reply:
@weshayutin Understood. Is it going to be fixed in OADP 1.2.0? 
If not, maybe we should not specify that it's a 1.1 issue at all. 
Once it's fixed, we can say this section is for OADP versions 1.2.z and before, with z being last z-stream to need the manual cleanups -- and end the section by saying "This issue was resolved in OADP 1.2.fixed-z-stream."